Classes/Domain/Collection/MetaDataCollection.php

```diff
@@ -13,7 +13,10 @@
 
 use Doctrine\Common\Collections\ArrayCollection;
 
+/**
+ * ArrayCollection for meta data
```

Comment:
Comments always tend to get outdated. So if a comment doesn't add any information, just leave it out.